content/java/concepts/classes/classes.md

```diff
@@ -15,31 +15,34 @@ CatalogContent:
   - 'paths/computer-science'
 ---
 
-A class is a blueprint or a template for an object. Blueprints detail the general structure (information and behavior) about an object.
+In Java, classes are blueprints or templates for objects in Java. They detail the general structure and data for an object including information such as properties, attributes, and method behavior.
 
 ## Syntax
 
-```java
-public class Cat {
-  String name;
-  String color;
-  int age;
+```pseudo
+accessModifier class ClassName {
+  dataType attributeOne;
+  dataType attributeTwo;
+  dataType attributeN;
 
-  public static void main(String[] args) {
-    Cat myCat = new Cat();
-
-    System.out.println("Name: " + myCat.name);
-    System.out.println("Color: " + myCat.color);
-    System.out.println("Age: " + myCat.age);
+  static void classMethod {
+    // Method code here.
   }
 }
 ```
 
-## Java Instance
+- Class names must always be in "TitleCase" and match the name the filename (e.g. `ClassName.java`).
```

Comment:
Is TitleCase and PascalCase interchangeable? I'm wondering if PascalCase is more standard

Reply:
Great question! I just did [a quick lookup](https://en.wikipedia.org/wiki/Camel_case#:~:text=Camel%20case%20is%20distinct,Ada%2095%20style%20guide.) and it looks like the two are actually different!

* "TitleCase" should actually be stylized as "Title Case" because it capitalizes each word while retaining the spaces.
* "PascalCase" is more accurate because Java class names should _**not**_ have spaces in them.

I really appreciate this catch! Thank you! #TIL